x-pack/plugins/security_solution/server/usage/endpoints/index.ts

```diff
@@ -51,16 +50,27 @@ export interface AgentLocalMetadata extends AgentMetadata {
   };
 }
 
-export type OSTracker = Record<string, AgentOSMetadataTelemetry>;
+type OSTracker = Record<string, AgentOSMetadataTelemetry>;
+type AgentDailyActiveTracker = Map<string, boolean>;
 /**
  * @description returns an empty telemetry object to be incrmented and updated within the `getEndpointTelemetryFromFleet` fn
  */
 export const getDefaultEndpointTelemetry = (): EndpointUsage => ({
   total_installed: 0,
   active_within_last_24_hours: 0,
   os: [],
+  policies: {
+    malware: {
+      active: 0,
+      inactive: 0,
+      failure: 0,
+    },
+  },
 });
 
+/**
+ * @description this fun
+ */
```

Comment:
This is fun? xD

Reply:
Yea, noticed after I pushed, but not going through CI again for this 😄 Will update in bug fixes